Makefile

```
@@ -33,7 +33,7 @@ test-php-integration: run-ocis-with-keycloak
 
 .PHONY: test-php-integration-ci
 test-php-integration-ci:            ## Run php integration tests in CI
-test-php-integration-ci: vendor/bin/phpunit
+	composer install
```

this was fine. or was there any issue with it?
```suggestion
test-php-integration-ci: vendor/bin/phpunit
```

---

I am trying to figure out what is causing cache not working